tests/testthat/test-propose_snapshot.R

```diff
@@ -49,7 +49,7 @@ test_that("propose_snapshot()", {
       "https://staging.r-multiverse.org/api/snapshot/zip",
       "?types=src,win,mac",
       "&binaries=",
-      gsub("\\.[0-9]*$", "", rversions::r_release()$version),
+      rversions::r_release()$version,
```

Comment:
Suggest to store `rversions::r_release()$version` as a variable upfront and supply this to the 'rversions' argument of 'propose_snapshot()' as well as here.

Reply:
done in 5124cd9dfdb34b81aafc27a077bcd4ac94a3b8a4